packages/discord.js/src/client/Client.js

```diff
@@ -258,7 +259,9 @@ class Client extends BaseClient {
   async login(token = this.token) {
     if (!token || typeof token !== 'string') throw new DiscordjsError(ErrorCodes.TokenInvalid);
     this.token = token = token.replace(/^(Bot|Bearer)\s*/i, '');
```

Comment:
Don't need to reassign the parameter anymore

```suggestion
    this.token = token.replace(/^(Bot|Bearer)\s*/i, '');
```